pkg/crc/version/version.go

```diff
@@ -91,7 +91,12 @@ func InstallPath() string {
 		logging.Errorf("Failed to find the installation path: %v", err)
 		return ""
 	}
-	return filepath.Dir(currentExecutablePath)
+	src, err := filepath.EvalSymlinks(currentExecutablePath)
+	if err != nil {
+		logging.Errorf("Failed to find the symlink destination of %s: %v", currentExecutablePath, err)
+		return ""
```

Comment:
might be better to return `filepath.Dir(currentExecutablepath)`?

Reply:
done!